gravatar-ui/src/main/java/com/gravatar/ui/components/ProfileSummary.kt

```diff
@@ -68,27 +70,18 @@ public fun ProfileSummary(
             EmptyProfileClickableContainer(state) {
                 Row(modifier = modifier) {
                     avatar(state)
-                    Column(modifier = Modifier.padding(start = 14.dp)) {
+                    Column(
+                        modifier = Modifier
+                            .padding(start = 14.dp)
+                            .align(Alignment.CenterVertically),
+                        verticalArrangement = Arrangement.Center,
+                    ) {
                         DisplayName(
                             state,
                             textStyle = MaterialTheme.typography.titleMedium.copy(fontWeight = FontWeight.Bold),
                             skeletonModifier = Modifier.fillMaxWidth(0.5f),
                         )
-                        when (state) {
-                            is ComponentState.Loaded -> {
-                                if (state.loadedValue.location.isNotBlank()) {
-                                    Location(state)
-                                }
-                            }
-
-                            ComponentState.Loading -> {
-                                Location(state = state, skeletonModifier = Modifier.fillMaxWidth(0.9f))
-                            }
-
-                            ComponentState.Empty -> {
-                                Location(state)
-                            }
-                        }
```

Comment:
This was pretty much a copy-paste of the `Location` component so I removed it from here. I guess this is a leftover.